website/docs/getting-started.md

```diff
@@ -228,7 +228,7 @@ npx lerna run test --scope=header
 ```
 
 ```
-> lerna run test --scope=header
+> npx lerna run test --scope=header
```

Comment:
The output in this case will be, it's just lerna that is echoing the command you were running.

```
> test
> lerna run test --scope=header
```

Reply:
I revert my change, thanks